app/services/report.py

```diff
@@ -29,7 +29,10 @@ def add_report(self, twitter_id: str, reporter_id: str) -> Report:
         """
         report_db = Report.objects(twitter_id=twitter_id).first()
         if report_db is None:
-            user = self.twitter_scraper.get_user_by_id(twitter_id)
+            user = TwitterUser.objects(twitter_id=twitter_id).first()
+
+            if user is None:
+                raise HTTPException(404, "Twitter account does not exist")
```

Comment:
so we not gonna include the case: re-scrape and save report when twitter user data is deleted?

Reply:
Because check result can change compared to result displayed on frontend.

Also see: https://github.com/dirtyg2120/thesis-backend/pull/25#discussion_r827597420

Reply:
Yea we're gonna update frontend to tell user to re-check instead

Reply:
kay, maybe tiêm mũi 3 make my memory worse